homeassistant/components/sensor/filter.py

```diff
@@ -20,16 +21,21 @@
 from homeassistant.util.decorator import Registry
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.event import async_track_state_change
+import homeassistant.components.history as history
+import homeassistant.util.dt as dt_util
 
 _LOGGER = logging.getLogger(__name__)
 
+DEPENDENCIES = ['history']
```

Comment:
We should not add this. Instead, check if history component is loaded inside the `added_to_hass` callback and only execute restore logic if it is. Home Assistant will make sure that if the user wants to enable the history component that it is set up prior to the sensor component.

Check like this:

```python
if not 'history' in self.hass.config.components:
    return
```